src/select/src/components/styled/selectStyles.js

```diff
@@ -0,0 +1,57 @@
+import { css } from 'styled-components';
+
+export default css`
+  .lattice-select__control {
+    @import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800");
```

Comment:
I believe we should be inheriting the page's font details. Are you sure this is necessary?